integration/Rakefile

```diff
@@ -1,13 +1,38 @@
 require 'rototiller'
 
+namespace :ci do
+  namespace :test do
+
+    desc 'Tests at the component level for the pe-r10k project'
+    task :component => [:check_client_tool_env_vars] do
+      Rake::Task[:beaker].invoke
+    end
+  end
+end
+
+desc 'Run tests against a packaged PE build'
+task :acceptance do
+  @acceptance_pre_suite = 'pre-suite'
+  Rake::Task[:beaker].invoke
+end
+
 desc 'The acceptance tests for r10k, run in the beaker framework'
-rototiller_task :acceptance => [:beaker_hostgenerator] do |t|
+rototiller_task :beaker => [:beaker_hostgenerator] do |t|
+
+  common_setup = <<-EOS
+pre-suite/00_pe_install.rb,
+component/pre-suite/05_install_dev_r10k.rb,
+pre-suite/10_git_config.rb,
+pre-suite/20_pe_r10k.rb,
+pre-suite/30_test_utils.rb,
+EOS
+  common_setup.gsub!("\n", '')
 
   flags = [
       {:name => '--hosts', :default => 'configs/generated', :override_env => 'BEAKER_HOST'},
       {:name => '--keyfile', :default => "#{ENV['HOME']}/.ssh/id_rsa-acceptance", :override_env => 'BEAKER_KEYFILE'},
       {:name => '--load-path', :default => 'lib'},
-      {:name => '--pre-suite', :default => 'pre-suite'},
+      {:name => '--pre-suite', :default => @acceptance_pre_suite || common_setup, :override_env => 'BEAKER_PRE_SUITE'},
```

Comment:
`BEAKER_PRE_SUITE` is total copy pasta.  And I'm unclear what it is and if it's what we want here.